assets/src/components/v2/dup/version.tsx

```diff
@@ -0,0 +1 @@
+export const DUP_VERSION = "21.4.6.1";
```

Comment:
I think we should bump the DUP version

Reply:
I agree, but the version is only used at the time of packaging. It is probably a good idea to create a PR when we package up the app that changes this string (that is what the last change to the v1 version file did but it wasn't done super consistently). That way, we can have a more reliable change log. What do you think?

Reply:
Ohhhh my bad. I forgot that's part of the packaging process. Cool, thanks!